packages/block-library/src/site-logo/editor.scss

```diff
@@ -139,6 +139,7 @@
 
 	img {
 		width: $grid-unit-50 * 0.5;
+		min-width: $grid-unit-50 * 0.5;
```

Comment:
We probably need a separate variable for this size. There's $radio-input-size, but it might be worth either creating another, or renaming that one so we have a single size for this. Can even be a new $grid-unit-25. But short of doing that, maybe it's fine to just type out `20px`?

Reply:
I'll trust your judgement about this and will update the code. If the new designs(or some current ones are using the `20px` value, I can create a new var. Alternatively I can hardcode the `20px` for now.